conda_smithy/configure_feedstock.py

```diff
@@ -524,7 +524,7 @@ def main(forge_file_directory):
     build_config.CONDA_PY = 10
 
     recipe_dir = 'recipe'
-    config = {'docker': {'image': 'condaforge/linux-anvil', 'command': 'bash'},
+    config = {'docker': {'image': 'condaforge/linux-anvil', 'command': 'bash -e'},
```

Comment:
Maybe we should just do `set -e` in `run_docker_build.tmpl` instead. Wouldn't want the container closing on me because I typoed something while debugging.